lighthouse-core/audits/resource-summary.js

```diff
@@ -68,24 +71,31 @@ class ResourceSummary extends Audit {
       'third-party': str_(i18n.UIStrings.thirdPartyResourceType),
     };
 
-    const types = /** @type {Array<LH.Budget.ResourceType>} */ (Object.keys(summary));
+    const types = /** @type {Array<Exclude<LH.Budget.ResourceType, 'third-party'>>} */
```

Comment:
does this change in type assertion have any effect here? just curious since it's unexpected to me :)

Reply:
It does - `summary[type].size` and `summary[type].count` will error without this change - and rightly so, since summary no longer includes a `third-party` property.